source/docs/software/frc-glossary.rst

```diff
@@ -0,0 +1,72 @@
+.. include:: <isonum.txt>
+
+FRC Glossary
+============
+
+.. glossary::
+
+   accelerometer
+      A common sensors used to measure acceleration in one or more axis.
+
+   alliance
+      A cooperative of up to four (4) FIRST\ |reg| Robotics Competition teams.
+
+   auto
+      The first phase of each :term:`match` is called Autonomous (auto) and consists of the first fifteen (0:15) seconds.
+
+   COTS
+      Commercial off the shelf, a standard (i.e. not custom order) part commonly available from a vendor to all teams for purchase.
+
+   C++
+      One of the three officially supported programming languages.
+
+   deprecated
+      Software that will be maintained for at least 1 year, but may be removed after that.
+
+   DHCP
+      Dynamic Host Configuration Protocol, the protocol that allows a central device to assign unique IP addresses to all other devices.
+
+   FMS
+      Field Management System, the electronics core responsible for sensing and controlling the FIRST Robotics Competition field.
+
+   GradleRIO
+      The mechanism that powers the deployment of robot code to the roboRIO.
+
+   gyroscope
+      A device that measures rate-of-rotation.  It can add up the rotation measurements to determine :term:`heading` of the robot. (“gyro”, for short)
```

Comment:
I don't think the hyphens are necessary here. Just "rate of rotation" should work.